File: json/src/main/java/jsimple/json/JsonParsingException.java
package jsimple.json;

/**
 * @author Bret Johnson
 * @since 5/6/12 2:13 AM
 */
public final class JsonParsingException extends JsonException {
    JsonParsingException(String expected, String encountered, Token token) {
        super("Expected " + expected + " but encountered " + encountered);
    }

    JsonParsingException(String message, Token token) {
        super(message);
    }
}

File: jsimple-json/c#/jsimple/json/JsonParsingException.cs
namespace jsimple.json
{

	/// <summary>
	/// @author Bret Johnson
	/// @since 5/6/12 2:13 AM
	/// </summary>
	public sealed class JsonParsingException : JsonException
	{
		internal JsonParsingException(string expected, string encountered, Token token) : base("Expected " + expected + " but encountered " + encountered)
		{
		}

		internal JsonParsingException(string message, Token token) : base(message)
		{
		}
	}

}